typings/index.d.ts

```diff
@@ -296,7 +296,8 @@ declare module 'discord.js' {
         AppIcon: (userID: string | number, hash: string, format: AllowedImageFormat, size: number) => string;
         AppAsset: (userID: string | number, hash: string, format: AllowedImageFormat, size: number) => string;
         GDMIcon: (userID: string | number, hash: string, format: AllowedImageFormat, size: number) => string;
-        Splash: (userID: string | number, hash: string, format: AllowedImageFormat, size: number) => string;
+        Splash: (guildID: string | number, hash: string, format: AllowedImageFormat, size: number) => string;
+        DisplaySplash: (guildID: string | number, hash: string, format: AllowedImageFormat, size: number) => string;
```

Comment:
Is this supposed to be DiscoverySplash?

Reply:
Oops, yeah, it is, I apologize, I've done that once before I pushed even (when I defined the function initially). Thanks for pointing it out, sorry for that